Crash creating file via ssh

```
05-13 17:35:18.653 3883-15463/? E/Surface: getSlotFromBufferLocked: unknown buffer: 0xb97ad888
05-13 17:35:18.661 7137-7282/com.amaze.filemanager E/SshClientUtils: Error executing template method
    net.schmizz.sshj.sftp.SFTPException: Permission denied
        at net.schmizz.sshj.sftp.Response.error(Response.java:137)
        at net.schmizz.sshj.sftp.Response.ensurePacketTypeIs(Response.java:114)
        at net.schmizz.sshj.sftp.SFTPEngine.open(SFTPEngine.java:143)
        at net.schmizz.sshj.sftp.SFTPClient.open(SFTPClient.java:68)
        at net.schmizz.sshj.sftp.SFTPClient.open(SFTPClient.java:73)
        at com.amaze.filemanager.filesystem.HybridFile$11.execute(HybridFile.java:1014)
        at com.amaze.filemanager.filesystem.HybridFile$11.execute(HybridFile.java:1010)
        at com.amaze.filemanager.filesystem.ssh.SshClientUtils.execute(SshClientUtils.java:73)
        at com.amaze.filemanager.filesystem.HybridFile.getOutputStream(HybridFile.java:1010)
        at com.amaze.filemanager.filesystem.Operations$2.doInBackground(Operations.java:215)
        at com.amaze.filemanager.filesystem.Operations$2.doInBackground(Operations.java:198)
        at android.os.AsyncTask$2.call(AsyncTask.java:295)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
        at java.lang.Thread.run(Thread.java:818)
05-13 17:35:18.662 7137-7282/com.amaze.filemanager E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #3
    Process: com.amaze.filemanager, PID: 7137
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:309)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
        at java.util.concurrent.FutureTask.run(FutureTask.java:242)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
        at java.lang.Thread.run(Thread.java:818)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void java.io.OutputStream.close()' on a null object reference
        at com.amaze.filemanager.filesystem.Operations$2.doInBackground(Operations.java:217)
        at com.amaze.filemanager.filesystem.Operations$2.doInBackground(Operations.java:198)
        at android.os.AsyncTask$2.call(AsyncTask.java:295)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588) 
        at java.lang.Thread.run(Thread.java:818) 
```

--- a/app/src/main/java/com/amaze/filemanager/filesystem/Operations.java
+++ b/app/src/main/java/com/amaze/filemanager/filesystem/Operations.java
@@ -213,6 +213,10 @@ public class Operations {
                 }
                 if (file.isSftp()) {
                     OutputStream out = file.getOutputStream(context);
+                    if(out == null) {
+                        errorCallBack.done(file, false);
+                        return null;
+                    }
                     try {
                         out.close();
                         errorCallBack.done(file, true);

